packages/gluestick/src/types.js

```diff
@@ -220,6 +220,8 @@ export type Hooks = {
   postGetCurrentRoute?: Hook;
   postRender?: Hook;
   error?: Hook;
+  webpackClientConfig?: Hook;
+  webpackServerConfig?: Hook;
```

Comment:
It'll never exist in this forms. What's the point to adding it here instead of creating new type called webpackHooks or something? Those plugins will not be used together in any case.